PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/SparkConsoleExecuteAction.kt

```diff
@@ -85,14 +89,29 @@ class SparkConsoleExecuteAction() : AzureAnAction(), DumbAware, ILogger {
 
         // Send to process as a whole codes block
         val normalizedCodes = text.trimEnd() + "\n"
-        try {
+        val codeHint = normalizedCodes.split("\n").first()
```

Comment:
The code might be only one line but with a long string. In this case the process bar will be messy. Maybe you can test this case first to see if it works well. If not, suggest to truncate the string to limited length.

Reply:
So far so good:
![spark_console_too_long_progress](https://user-images.githubusercontent.com/7385667/67262813-a8959780-f4d8-11e9-8fa1-e7a540e3c4c4.gif)